BUILD.gn

```diff
@@ -485,12 +485,12 @@ static_library("electron_lib") {
       "//chrome/renderer/prerender/prerender_helper.h",
       "//chrome/renderer/prerender/prerenderer_client.cc",
       "//chrome/renderer/prerender/prerenderer_client.h",
-      "//chrome/utility/printing_handler.cc",
-      "//chrome/utility/printing_handler.h",
     ]
 
     if (is_win) {
       sources += [
+        "//chrome/utility/printing_handler.cc",
+        "//chrome/utility/printing_handler.h",
```

Comment:
@brenca `sources` should be sorted alphabetically, like any other lists in GN configs.
The linter job didn't report its results for some reason.

Reply:
I had to fix the linter, it refused to run for me on Windows, thanks for catching this!